specification/marketplacecatalog/resource-manager/Search/stable/2022-02-02/search.json

```diff
@@ -0,0 +1,785 @@
+          "items": {
+            "type": "string"
+          }
+        },
+        "azurePortalCategories": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "azureIndustries": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "bigId": {
+          "type": "string"
+        },
+        "cspStates": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "determinedStorefronts": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "displayName": {
+          "type": "string"
+        },
+        "isAzureBenefitEligible": {
+          "type": "boolean"
+        },
+        "isCoreVm": {
+          "type": "boolean"
+        },
+        "isPreferredSolution": {
+          "type": "boolean"
+        },
+        "isAdditionalPurchaseRequired": {
+          "type": "boolean"
+        },
+        "isPowerBICertified": {
+          "type": "boolean"
+        },
+        "isIndustryCloud": {
+          "type": "boolean"
+        },
+        "isMicrosoftProduct": {
+          "type": "boolean"
+        },
+        "isPreview": {
+          "type": "boolean"
+        },
+        "language": {
+          "type": "string"
+        },
+        "legacyId": {
+          "type": "string"
+        },
+        "offerType": {
+          "type": "string"
+        },
+        "operatingSystems": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "plans": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PlanDto"
+          },
+          "x-ms-identifiers": [
+            "legacyPlanId"
+          ]
+        },
+        "pricingTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "publisherDisplayName": {
+          "type": "string"
+        },
+        "summary": {
+          "type": "string"
+        },
+        "vmGenerations": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "vmImageTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "vmSecurityTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "smallIconUri": {
+          "type": "string"
+        },
+        "mediumIconUri": {
+          "type": "string"
+        },
+        "largeIconUri": {
+          "type": "string"
+        },
+        "wideIconUri": {
+          "type": "string"
+        },
+        "heroIconUri": {
+          "type": "string"
+        }
+      }
+    },
+    "PlanDto": {
+      "type": "object",
+      "properties": {
+        "cspState": {
```

Comment:
>cspState [](http://example.com/codeflow?start=9&length=8)

Consider expanding the acronym in the name.

Reply:
@mentat9 I understand your point but unfortunately this property is based on cosmosDB property that has similar name so I would want to keep the same naming and type for this field so it wont confuse our clients that use both Search API and other API's that get data from our cosmos.